dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.audit-1.0.javaee.feature

```diff
@@ -0,0 +1,16 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+
+symbolicName = io.openliberty.audit-1.0.javaee
+visibility = private
+
+IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.audit-1.0))",\
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=com.ibm.websphere.appserver.servlet-3.0)(osgi.identity=com.ibm.websphere.appserver.servlet-3.1)(osgi.identity=com.ibm.websphere.appserver.servlet-4.0)))"
+
+IBM-Install-Policy: when-satisfied
+
+-features=\
+  com.ibm.websphere.appserver.appSecurity-2.0; ibm.tolerates:="3.0", \
+  com.ibm.websphere.appserver.auditCollector-1.0
```

Comment:
Remove the auditCollector-1.0 feature. We will leave it in the public feature.

Reply:
It is also possible that you may need to put a dependency on the servlet-3.0 feature (tolerating 3.1 and 4.0).